Cell.xctemplate/___FILEBASENAME___Cell.swift

```diff
@@ -0,0 +1,25 @@
+//
+//  ___FILENAME___
+//  ___PROJECTNAME___
+//
+//  Created by ___FULLUSERNAME___ on ___DATE___.
+//  Copyright © ___YEAR___ ___ORGANIZATIONNAME___. All rights reserved.
+//
+
+import FuntastyKit
+
+final class ___FILEBASENAMEASIDENTIFIER___Cell: UITableViewCell {
```

Comment:
Stále je potřeba upravit pro Xcode 9. `___FILEBASENAMEASIDENTIFIER___` vezme celý název souboru, který už např. `Cell` v sobě obsahuje, tak se to zduplikuje. Na to si můžeš použít tu proměnnou z plistu, kterou sis tam nadefinoval, například v novývh MVVM šablonách to je `___VARIABLE_sceneIdentifier___`. Zkontroluj si, z které verze šablon vycházíš.